lib/error.php

```diff
@@ -79,3 +79,34 @@ function logBridgeError($bridgeName, $code)
 
     return $report['count'];
 }
+
+function create_sane_stacktrace(\Throwable $e): array
+{
+    $frames = array_reverse($e->getTrace());
+    $frames[] = [
+        'file' => $e->getFile(),
+        'line' => $e->getLine(),
+    ];
+    $stackTrace = [];
+    foreach ($frames as $i => $frame) {
+        $file = $frame['file'] ?? '(no file)';
+        $line = $frame['line'] ?? '(no line)';
+        $stackTrace[] = sprintf(
+            '#%s %s:%s',
```

Comment:
It's a good idea but would have to recreate it when using the stacktrace in the github issue body. Also I don't think `<ol>` supports starting with zero.

Reply:
We are computer people. We start counting at zero :D

I chose to start counting at zero because php's default error handler also starts at zero.

Reply:
When there are different representations, it makes even more sense to keep the source data in universal form and render them as needed with `stacktrace_to_html` & `stacktrace_to_markdown`. E.g. for the HTML page, it might be useful to add the link to the files – see how [Tracy](https://tracy.nette.org/) does it: https://nette.github.io/tracy/tracy-exception.html